testing/e2e/skr-tester/pkg/command/update.go

```diff
@@ -95,21 +98,56 @@ func (cmd *UpdateCommand) Run() error {
 							return fmt.Errorf("error updating instance: %v", err)
 						}
 						fmt.Printf("Update operationID: %s\n", resp["operation"].(string))
-						break
+						return nil
 					}
 				}
+
 			}
 		}
+	} else if cmd.updateOIDC {
+		currentOIDCConfig, err := kcpClient.GetCurrentOIDCConfig(cmd.instanceID)
+		if err != nil {
+			return fmt.Errorf("failed to get current OIDC config: %v", err)
+		}
+		fmt.Printf("Current OIDC config: %v\n", currentOIDCConfig)
+		newOIDCConfig := map[string]interface{}{
+			"clientID":       "foo-bar",
+			"groupsClaim":    "groups1",
+			"issuerURL":      "https://new.custom.ias.com",
+			"signingAlgs":    []string{"RS256"},
+			"usernameClaim":  "email",
+			"usernamePrefix": "acme-",
+		}
+		fmt.Printf("Determined OIDC configuration to update: %v\n", newOIDCConfig)
+		resp, err := brokerClient.UpdateInstance(cmd.instanceID, map[string]interface{}{"oidc": newOIDCConfig})
+		if err != nil {
+			return fmt.Errorf("error updating instance: %v", err)
+		}
+		fmt.Printf("Update operationID: %s\n", resp["operation"].(string))
+	} else if cmd.updateAdministrators {
+		//TODO print current admnistrators
```

Comment:
This will be implemented along with the assertions, as we need to connect to the SKR to get admins.